src/main/java/org/desarrolladorslp/technovation/controller/MessageController.java

```diff
@@ -0,0 +1,52 @@
+package org.desarrolladorslp.technovation.controller;
+
+import com.google.gson.internal.LinkedTreeMap;
+import org.desarrolladorslp.technovation.config.auth.TokenInfoService;
+import org.desarrolladorslp.technovation.controller.dto.MessageHeaderDTO;
+import org.desarrolladorslp.technovation.models.Message;
+import org.desarrolladorslp.technovation.services.MessageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+import java.util.*;
+
+@RestController
+@RequestMapping("api/message")
+public class MessageController {
+
+
+    private MessageService messageService;
+
+    private TokenInfoService tokenInfoService;
+
+
+    @RequestMapping("/send")
+    @PostMapping
+    public ResponseEntity<Message> sendMessage(@RequestBody LinkedTreeMap obj, Principal principal){
```

Comment:
Create a DTO to receive the information about the message to be sent. 
Use descriptive names for the variables `obj`